### PR TITLE
update reposense claims, update ug. fix edit trip bug, new trip tests

### PR DESCRIPTION
**Pull request by yeezao**

### CODE REVIEW

**Review by itsleeqian — APPROVED** [review]
lgtm

File: src/main/java/seedu/duke/parser/CommandExecutor.java [hunk 1]
```diff
@@ -362,7 +363,7 @@ private static void executeDeleteTrip(int tripIndex) {
      * @param attributeToEdit String of all attributes to be added and their new values
      */
     private static void editTripPerAttribute(Trip tripToEdit, String attributeToEdit) throws ForceCancelException {
-        String[] splitCommandAndData = attributeToEdit.split(" ");
+        String[] splitCommandAndData = attributeToEdit.split(" ", 2);
```
Comment (leeyikai) [inline src/main/java/seedu/duke/parser/CommandExecutor.java:366]: You might want to consider removing magic numbers here

**Review by leeyikai — APPROVED** [review]
Small changes but LGTM!

File: src/main/java/seedu/duke/parser/CommandExecutor.java (hunk 1 above)
Comment (yeezao) [reply src/main/java/seedu/duke/parser/CommandExecutor.java:366]: thanks for the spot! 😎

**Pull request merged**